.cargo/config.toml

```diff
@@ -1,5 +1,58 @@
-# Deployment runtime lints
-[target.'cfg(all(target_arch = "wasm32", not(test)))']
+# Unfortunately we cannot add any lints based on `cfg(test)` nor `cfg(feature = "feat")`: https://github.com/rust-lang/cargo/issues/8170.
+# Hence, we can only define lints that are applied codebase-wide (including tests), and lints that are applied only to runtime code.
+
+# Codebase-wide lints.
+[target.'cfg(all())']
+rustflags = [
```

Comment:
No strong opinion on that but I thing, we have some single match statements: https://rust-lang.github.io/rust-clippy/master/index.html#single_match

Reply:
I'm not sure this is a good idea. We would fail to catch if the enum has new variants, and we would simply fail. I think having explicit enumeration is almost always better.